libs/features/pages/consultancy-page/src/lib/components/elewa-website-learning-services/elewa-website-learning-services.component.scss

```diff
@@ -0,0 +1,61 @@
+.learning-services{
+    align-items: center;
+    flex-direction: column;
+    display: flex;
+
+    .container {
+        text-align: center; 
+        height: 100px; 
+        
+        h1 {
+            display: inline-block; 
+            vertical-align: middle; 
+        }
+    
+        span {
+            display: block;
+        }
+    }
+
+    .cards { 
+        position: relative;
+        gap: 20px; 
+        padding: 12px;
+        max-width: 100%;
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+    }
+
+    .evaluate{
+        position: absolute;
+        top: 494px;
+        right: 200px;
+    }
```

Comment:
use display grid to position the items. position absolute will not be the best approach when considering responsiveness.